package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>sophus</name>
+  <version>1.0.0</version>
+  <description>
+   C++ implementation of Lie Groups using Eigen.
+  </description>
+  <url type="repository">https://github.com/stonier/sophus</url>
+  <url type="bugtracker">https://github.com/stonier/sophus/issues</url>
+  <maintainer email="nomail@nomail.com">Hauke Strasdat</maintainer>
+  <author>Hauke Strasdat</author>
+  <license>BSD</license>
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <build_depend>eigen</build_depend>
```

Comment:
Shouldn't there also be a `build_export_depend` on `eigen`, since you need eigen to build stuff that uses Sophus. That means we could use a single `depend` tag (if anything, maybe `exec_depend` is not needed, since there is no shared library or executable in sophus that depends on a shared library or executable in eigen, or is there).